scripts/release/debian/build.sh

```diff
@@ -42,7 +42,7 @@ wget -qO- https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSI
 echo "Python source code is in $PYTHON_SRC_DIR"
 
 # Build Python
-$PYTHON_SRC_DIR/*/configure --srcdir $PYTHON_SRC_DIR/* --prefix $WORKDIR/python_env
+$PYTHON_SRC_DIR/*/configure --srcdir $PYTHON_SRC_DIR/* --prefix $WORKDIR/python_env --disable-test-modules
```

Comment:
Any official doc for `--disable-test-modules`?

Reply:
> Don’t build nor install test modules, like the [test](https://docs.python.org/3/library/test.html#module-test) package or the _testcapi extension module (built and installed by default).
https://docs.python.org/3/using/configure.html#cmdoption-disable-test-modules